ci-operator/step-registry/rosa/cluster/provision/rosa-cluster-provision-commands.sh

```diff
@@ -3,6 +3,7 @@
 set -o nounset
 set -o errexit
 set -o pipefail
+set -x
```

Comment:
I think `set -x` generally used for debug/development.  Is it expected that it's still there given the steps are already verified ?

Reply:
Thanks, have removed